app/src/main/java/cm/aptoide/pt/download/DownloadAnalytics.java

```diff
@@ -266,7 +266,7 @@ public void installClicked(String md5, String packageName, String trustedValue,
       downloadCompleteEvent(navigationTracker.getPreviousScreen(),
           navigationTracker.getCurrentScreen(), md5, packageName, trustedValue, action,
           previousContext,
-          offerResponseStatus.equals(WalletAdsOfferManager.OfferResponseStatus.ADS_SHOW));
+          offerResponseStatus.equals(WalletAdsOfferManager.OfferResponseStatus.ADS_HIDE));
```

Comment:
does this make sense with the condition?` if (!offerResponseStatus.equals(WalletAdsOfferManager.OfferResponseStatus.NO_ADS)) {
`

Reply:
yes, because NO_ADS means that there are no ads (example ab test), so the parameter should not be added to the download complete event.